hikari/impl/rest.py

```diff
@@ -1424,6 +1426,9 @@ def _build_message_payload(  # noqa: C901- Function too complex
         if not undefined.any_undefined(sticker, stickers):
             raise ValueError("You may only specify one of 'sticker' or 'stickers', not both")
 
+        if not undefined.all_undefined(waveform, duration) and undefined.any_undefined(waveform, duration):
+            raise ValueError("You may only specify both of 'waveform' and 'duration', not only one")
```

Comment:
```suggestion
            raise ValueError("You need to specify both 'waveform' and 'duration', not only one")
```

Im also tempted to just not have it and leave it up to Discord to validate it

Reply:
I'd say just let Discord handle it, personally.

Reply:
this will be taken out of `_build_message_payload` anyways and in the `create_voice_message` its a required argument so we don't need these checks anymore